README.md

```diff
@@ -1,12 +1,14 @@
 [![Build Status](https://travis-ci.org/OCA/purchase-workflow.svg?branch=7.0)](https://travis-ci.org/OCA/purchase-workflow)
```

Comment:
:) I agree. Name of the tools would make more sense and be coherent with the other badges, but honestly, this is not even a cosmetic issue as it is an alternative to the image. I won't sleep any better if it changes.